default/stringtables/en.txt

```diff
@@ -4826,7 +4822,9 @@ DAMAGE_TITLE
 Damage
 
 DAMAGE_TEXT
-Defines the base extent that a weapon will reduce the [[metertype METER_STRUCTURE]] of another ship (or the [[metertype METER_SHIELD]], [[metertype METER_DEFENSE]] and [[metertype METER_CONSTRUCTION]] of a planet), in one combat round (of which there are three in each turn of battle).  If a target ship is equipped with [[metertype METER_SHIELD]], the actual damage caused by each hit will be the base damage reduced by the strength rating of the ship's [[metertype METER_SHIELD]].
+'''Defines the base extent that a weapon will reduce the [[metertype METER_STRUCTURE]] of another ship (or the [[metertype METER_SHIELD]], [[metertype METER_DEFENSE]] and [[metertype METER_CONSTRUCTION]] of a planet), in one combat round (of which there are three in each turn of battle).  If a target ship is equipped with [[metertype METER_SHIELD]], the actual damage caused by each hit will be the base damage reduced by the strength rating of the ship's [[metertype METER_SHIELD]].
+
+Note: [[encyclopedia FIGHTER_TECHS]] inflict damage ignoring [[metertype METER_SHIELD]] on warships, as fighters fire from within the enemy shields.'''
```

Comment:
Yes, my bad for all unrelated changes in only one commit... But they are minor changes (and not so much), so I thought it would be better to have all in one commit, and an easier check for the reviewers, rather than 4 or 5 commits.

Oh... and I can't deny I'm a  lazy man. :p